ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md

```diff
@@ -81,3 +82,49 @@ Where:
 * you can check the mapping between partition IDs and partition names using `gdisk -l` command
 
 If you insert the SD card into the DragonBoard410c and set the switch S6-2 (on the back) to ON, you should now be able to boot from SD card.
+
+# 4) Booting Debian image
+
+Download boot and root images from:
+  - **Release:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+    - Boot File: boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-XXX.img.gz
+    - Root File: 	linaro-stretch-alip-qcom-snapdragon-arm64-XXX.img.gz
+
+
+  - **Snapshot:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+    - Boot File: boot-sdcard-linaro-buster-dragonboard-410c-XXX.img.gz
+    - Root File: linaro-buster-alip-dragonboard-410c-XXX.img.gz
+
+
+  For Example:
+  ```shell
+  $ wget http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-20171016-283.img.gz
+  $ wget http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/linaro-stretch-alip-qcom-snapdragon-arm64-20171016-283.img.gz
+  ```
+  And then extract the images
+  ```shell
+  $ gunzip boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-20171016-283.img.gz
+  $ gunzip linaro-stretch-alip-qcom-snapdragon-arm64-20171016-283.img.gz
+  ```
+
+Since we are already downloading the boot-sdcard-xxx.img.gz, we do not need to modify the boot parameters.
+
+You can now write the boot and rootfs image into the SD card:
+
+```shell
+$ sudo dd if=boot-sdcard-XXX.img of=/dev/XXX7
+$ sudo simg2img linaro-stretch-alip-XXX.img /dev/XXX9
+```
+NOTE: you may need to install the ```simg2img``` package using:
+```shell
+$ sudo apt install simg2img
+# OR
+$ sudo apt install android-tools-fsutils
+```
+
+Where:
+* /dev/XXX7 represents the 'boot' partition on the SD card
+* /dev/XXX9 represents the 'rootfs' partition on the SD card
+* you can check the mapping between partition IDs and partition names using `gdisk -l` command
+
+If you insert the SD card into the DragonBoard410c and set the switch S6-2 (on the back) to ON, you should now be able to boot from SD card.
```

Comment:
Could be

Now insert the SD card into Dragonboard410c and set switch S6-2 (on back side) to ON. Then power ON the Dragonboard410c to see Debian booting from SD card.

Reply:
@Mani-Sadhasivam i'll modify the previous wget in a separate PR, and the issue with wget not accepting wildcards in http seems to be a recent one, afaik used to work before.

I'll modify the last line